tests/nexus_api_test.go

```diff
@@ -688,6 +690,9 @@ func (s *ClientFunctionalSuite) versionedNexusTaskPoller(ctx context.Context, ta
 	if err != nil {
 		panic(err)
 	}
+	if res == nil {
+		return
+	}
```

Comment:
In what situation will this be nil while an error is also nil? That seems like a bug

Reply:
Oh was debugging something, meant to remove that